build/Gruntfile.coffee

```diff
@@ -31,39 +31,52 @@ module.exports = (grunt) ->
   # This allows all subsequent paths to the relative to the root of the repo
   grunt.file.setBase(path.resolve('..'))
 
-  tmpDir = os.tmpdir()
-  appName = if process.platform is 'darwin' then 'Atom.app' else 'Atom'
```

Comment:
This change is what broke `master` on OS X since this means we end up with an incorrect `shellAppDir`.